workflow/scripts/build_target_regions.py

```diff
@@ -0,0 +1,20 @@
+log_file = open(snakemake.log[0], "w")
+out = open(snakemake.output[0], "w")
+
+with open(snakemake.input[0], "r") as primer_file:
+    for line in primer_file.readlines():
```

Comment:
```suggestion
    for line in primer_file:
```

Reply:
Better do this with pandas. Since both input and output are tab separated, should be two or three lines just. The input file could be large, but pandas can easily work on chunks:
https://stackoverflow.com/a/25962187/7070491

Reply:
Done!
While pandas comes along with snakemake should an environment be added to the rule nevertheless?